plugin/src/main/kotlin/spp/jetbrains/sourcemarker/status/util/AutocompleteField.kt

```diff
@@ -88,14 +91,19 @@ class AutocompleteField(
         addKeyListener(this)
 
         val sb = StringBuilder("(")
+        val sbt = StringBuilder("(")
```

Comment:
This logic looks like it belong in VariablePattern too.

Reply:
working on it. but again IDE is failing to connect to Source++. Need help

Reply:
resolved